### DISCUSSION TITLE
Set utility classes to change via breakpoint

### DISCUSSION THREAD

**Discussion by AlanBreck** [tailwindlabs/tailwindcss#11397] [General]
Is there a way of setting things up such that a utility class like `text-h1` changes in value based upon a breakpoint? Since I always know that I want my `h1`s to change size when on mobile, I'd rather not have to add something like `text-h2 sm:text-h1` to every `h1` tag.

**Comment by gukj** [comment]
What I've done for some projects is to create my own set of headers while leaving the `text-[size]` classes be for if/when I need them.

So something like:
```
.header-1 {
  @apply text-lg sm:text-xl md:text-2xl lg:text-3xl;
}
```

**Reply by AlanBreck** [reply]
That makes a lot of sense.

**Comment by mybearworld** [comment]
It sounds like you might need the [Prose plugin](https://tailwindcss.com/docs/typography-plugin).
